recipes/LibriSpeech/ASR/ctc/hparams/train_with_wav2vec.yaml

```diff
@@ -42,7 +42,7 @@ sample_rate: 16000
 # With data_parallel batch_size is split into N jobs
 # With DDP batch_size is multiplied by N jobs
 # Must be 6 per GPU to fit 16GB of VRAM
-batch_size: 8
+batch_size: 6
```

Comment:
Did you try with 6, because I am doing some VRAM tests right now, and it's crazy. BS has to be one to fit a 24GB GPU xD

Reply:
I used 8 for training and got OOM around last few iterations.